extensions/linear/src/components/IssueActions/IssueActions.tsx

```diff
@@ -421,6 +450,8 @@ export default function IssueActions({
           onChange={setDueDate}
         />
 
+        <Action.PickDate title="Set Reminder" shortcut={{ modifiers: ["cmd"], key: "h" }} onChange={setReminder} />
```

Comment:
Also, it would be nice that `Action.PickDate` allows to pick hours to reflect Linear's experience.

Reply:
Looks like the `Action.PickDate` doesn't expose the `type` of the `Form.DatePicker`. We should probably add it. I'll create an issue for it internally.